### ISSUE TITLE
Custom rules added

### ISSUE THREAD

**Issue by CodeCruncher24** [hagezi/dns-blocklists#4119]
### Which AdBlocker/DNS cloud service do you use?

ControlD

### Other

I have multiple custom rules added -- and not block list enabled

### ControlD users

- [X] IMPORTANT - I assure that I have not set the Block Response in ControlD to Custom or Branded and can confirm that the problem still occurs.

### NextDNS users

- [ ] IMPORTANT - I can assure that I disabled the block page in NextDNS and can confirm that the problem still occurs.

### With which block list(s) does the problem occur?

Native Tracker

### Which domain(s) should be unblocked?

agoda.com
trip.com

### Why should the domain(s) be unblocked?

seems a false positive due to some custom rule I am unable to figure out
I have added all of native tracker folders along with: spam tlds, spam idns, spam tlds (allow), malicious cc, Ad'n'Track Wildcard, analytics (both allow and block)
Currently I only have TIF, phishing and malware lists enabled.

### Privacy

- [X] I confirm that the report does not contain any private information.

**Comment by CodeCruncher24** [comment]
I have removed spam TLDs (both folders) -- trip and agoda both open now.
But I do get issues sometimes with gmail.com and similar google services where pages do not open. Is this affected by native trackers, analytics and adsntrack wildcard?

**Comment by hagezi** [comment]
@CodeCruncher24 I cannot reproduce any of the problems mentioned here. This is not a blocklist or folder problem ...

**Comment by CodeCruncher24** [comment]
Also, I feel like malicious CCs folder is causing issues with loading mainstream domains such as: gmail.com and nishatlinen.com
These are getting blocked may be because of my dns query traceroute -- I think

**Comment by hagezi** [comment]
Yes, you can block entire geo regions with it, take a look at the documentation in ControlD.
See: https://docs.controld.com/docs/geo-custom-rules

I will remove the problematic folders as they seem to be used without knowing what is being used.

**Comment by CodeCruncher24** [comment]
Or you could add a disclaimer or some note as this is powerful to use if done correctly, I just realized.
